### PR TITLE
LabView Client Examples Improvements

### LINKED ISSUE #72: Update LabView Client examples page with better descriptions and improved figures.
Expand the descriptions on the monoDrive LabView client examples page to better describe the purpose for each example. Elaborate on the controls and provide updated figures with better labels.

### PR DESCRIPTION
**Pull request by bradleynull**
This updates all of the LabView Client examples in the documentation. I've included better screenshots of each example with callouts for each of the new controls as your progress down the page. All of the figures are currently living in a [Google Slide](https://docs.google.com/presentation/d/1-0QpgdqyyD2dJ5oVlPwkN2V5RkzLl6Xho0pdjdX2Vdw/edit#slide=id.g72ff208c5e_4_23) in order to keep stylization the same and make updating the figures much easier. 

All descriptions have been updated as well and "steps" have been converted to features with bullets because I felt that made more sense. Each example has a blurb about the example, what you can do prior to running the example, and what you can do after the example is running.

Closes #72

### CODE REVIEW

File: docs/LV_client/quick_start/LabVIEW_run_examples.md [hunk 1]
```diff
@@ -1,128 +1,212 @@
...
 
 <div class="img_container">
-    <img class='wide_img' src="https://github.com/monoDriveIO/documentation/raw/master/docs/LV_client/quick_start_img/CL_example.png"/>
+    <img class='wide_img' src="../imgs/CL_example.png" />
 </div>
 
-<p>&nbsp;</p>
+Prior to running the example, the user can:
 
+* **Select the weather profile by changing selected weather profile using th
```
Comment (citlaligm) [inline docs/LV_client/quick_start/LabVIEW_run_examples.md:25]: typo:
 using the*

File: docs/LV_client/quick_start/LabVIEW_run_examples.md [hunk 2]
```diff
@@ -1,128 +1,212 @@
...
+in the monoDrive Simulator. 
 
+* **Select the map by using the Map Selection Drop-down.** When ran, the 
+monoDrive simulator will switch to this map prior to beginning the simulation.
 
-### Replay Example
-
-1. Click on the arrow on the top-left corner to start the client.
+* **Select a different position using the Start Position Control.** monoDrive 
+maps come with various different start positions around the map. The number in
```
Comment (citlaligm) [inline docs/LV_client/quick_start/LabVIEW_run_examples.md:33]: is it redundant "various" and "different"?

File: docs/LV_client/quick_start/LabVIEW_run_examples.md (hunk 1 above)
Comment (bradleynull) [reply docs/LV_client/quick_start/LabVIEW_run_examples.md:25]: Fixed

File: docs/LV_client/quick_start/LabVIEW_run_examples.md (hunk 2 above)
Comment (bradleynull) [reply docs/LV_client/quick_start/LabVIEW_run_examples.md:33]: Removed.

**Comment by bradleynull** [comment]
@citlaligm Addressed your PR comments. Ready for another look. Thanks!

**Pull request merged**